### PR TITLE
Fix chxtools handler

### PR DESCRIPTION
**Pull request by jrmlhermitte**
Changes made to chxtools. Mostly removed handlers and import from eiger_io
works in Yugang's notebook

WIP do not merge (some cleanup to do)

### CODE REVIEW

File: chxtools/handlers.py [hunk 1]
```diff
@@ -1,68 +1,19 @@
...
-    'x_pixel_size': 'entry/instrument/detector/x_pixel_size',
-    'detector_distance': 'entry/instrument/detector/detector_distance',
-    'incident_wavelength': 'entry/instrument/beam/incident_wavelength',
-    'frame_time': 'entry/instrument/detector/frame_time',
-    'beam_center_x': 'entry/instrument/detector/beam_center_x',
-    'beam_center_y': 'entry/instrument/detector/beam_center_y',
-    'count_time': 'entry/instrument/detector/count_time',
-    'pixel_mask': 'entry/instrument/detector/detectorSpecific/pixel_mask',
-}
+print('here')
```
Comment (mrakitin) [inline chxtools/handlers.py]: Do we need it?

File: chxtools/handlers.py [hunk 2]
```diff
@@ -1,68 +1,19 @@
...
-        pixel_mask[pixel_mask==2] = 1
-        md['framerate'] = 1./md['frame_time']
-        # TODO Return a multi-dimensional PIMS seq.
-        return FixedEigerImages(master_path, md)
-
-#db.fs.deregister_handler('AD_EIGER')
 db = Broker.named('chx')
-db.reg.register_handler('AD_EIGER', LazyEigerHandler)
+db.reg.register_handler('AD_EIGER2', EigerHandler)
+db.reg.register_handler('AD_EIGER2', EigerHandler)
```
Comment (mrakitin) [inline chxtools/handlers.py]: Why is it twice here?

File: chxtools/handlers.py (hunk 1 above)
Comment (jrmlhermitte) [reply chxtools/handlers.py]: nope. thanks!

File: chxtools/handlers.py (hunk 2 above)
Comment (jrmlhermitte) [reply chxtools/handlers.py]: thanks for catching this! fixed!

**Comment by jrmlhermitte** [comment]
put all handlers in one place (eiger io) and import from one place.
Fixed a few issues involving the new version of databroker (with some changes by @mrakitin and @danielballan )

The dask handler worked in every case except for parallel compression of files. So I have hard coded the code to use the PIMS handler for now.

ready for review

**Comment by mrakitin** [comment]
Looks great to me!

**Pull request merged**